website/docs/docs/dbt-versions/core-upgrade/06-upgrading-to-v1.9.md

```diff
@@ -107,6 +108,6 @@ You can read more about each of these behavior changes in the following links:
 We also made some quality-of-life improvements in Core 1.9, enabling you to:
 
 - Maintain data quality now that dbt returns an an error (versioned models) or warning (unversioned models) when someone [removes a contracted model by deleting, renaming, or disabling](/docs/collaborate/govern/model-contracts#how-are-breaking-changes-handled) it.
-- Document [singular data tests](/docs/build/data-tests#singular-data-tests).
+- Document [data tests](/docs/build/data-tests#singular-data-tests).
```

Comment:
Can we link to here instead? https://docs.getdbt.com/reference/resource-properties/description

Reply:
oh great catch, sorry i didn't catch that sooner!